pkg/segment/writer/startree_test.go

```diff
@@ -388,11 +388,11 @@ func TestStarTreeMedium(t *testing.T) {
 
 		// first TotalMeasFns will be for col "e"
 		agSumIdx := 1*(TotalMeasFns) + MeasFnSumIdx
-
-		assert.Equal(t, root.aggValues[agSumIdx].CVal.(int64),
+		fmt.Println("root.aggValues[agSumIdx]=", root.aggValues[agSumIdx])
+		assert.Equal(t, root.aggValues[agSumIdx].IntgrVal,
```

Comment:
nit: do we want this print in this test, or did you forget to remove it?

Reply:
I forgot to remove it.